frontend/redux/selectors.js

```diff
@@ -24,3 +24,22 @@ export const isLoading = state => Object
 
 export const demographics = state => state.demographics.questions;
 export const demographicsOrder = state => state.demographics.order;
+
+export const selectWorkload = function(state) {
+  const images = [];
+  const answers = store.getState().demographics.answers;
```

Comment:
`store` doesn't look like it will be defined here, I'd expect that this should be
```js
const answers = state.demographics.answers;
```
That this is working as-is results from `store` being available on `window.store`, which is debugging code only; and this would break once we remove that window assignment.